apps/ledger-live-mobile/src/screens/Settings/Debug/Features/Lottie.tsx

```diff
@@ -101,66 +93,52 @@ const DebugLottie = () => {
       ]}
     >
       <LText secondary semiBold style={styles.title}>
-        {!key ? "Select Animation" : `Showing '${key}'`}
+        {!selectedKey ? "Select Animation" : `Showing '${selectedKey}'`}
       </LText>
       <Flex flex={1}>
-        <ScrollView>
-          <View
-            style={{
-              borderWidth: 1,
-            }}
-          >
-            {animationLight && <Animation key={animationNodeKey} source={animationLight} />}
-          </View>
-          <View
-            style={{
-              backgroundColor: "#121212",
-            }}
-          >
-            {animationDark && <Animation key={animationNodeKey} source={animationDark} />}
-          </View>
+        <ScrollView style={{ backgroundColor: "grey" }}>
+          <Flex>
+            {animationLight && (
+              <Animation
+                key={animationNodeKey}
+                source={animationLight}
+                style={{
+                  backgroundColor: "white",
+                }}
+              />
+            )}
+          </Flex>
+          <Flex p={2}>
+            {animationDark && (
+              <Animation
+                key={animationNodeKey}
+                source={animationDark}
+                style={{
+                  backgroundColor: "#121212",
```

Comment:
Same as above can we use the DS colors ?

Reply:
I've updated the two places that you mentioned, but there is still many places to be updated and this is out of the subject of this ticket, maybe a dedicated ticket should be created.